### PR TITLE
[PBW-4493] added ASSET_DIMENSION constant

### PR DESCRIPTION
**Pull request by rchalooyala**

### CODE REVIEW

**Comment by rchalooyala** [comment]
bump

File: js/utils/constants.js [hunk 1]
```diff
@@ -306,6 +306,22 @@
...
+       * Provide the following arguments:
+       * <ul>
+       *   <li>width
+       *   </li>
+       *   <li>height
+       *   </li>
+       * </ul>
+       *
+       * @event OO.EVENTS#ASSET_DIMENSION
+       * @private
```
Comment (angieisrani) [inline js/utils/constants.js]: this looks public?

File: js/utils/constants.js [hunk 2]
```diff
@@ -306,6 +306,22 @@
       SET_CLOSED_CAPTIONS_LANGUAGE: 'setClosedCaptionsLanguage',
       CLOSED_CAPTION_CUE_CHANGED: 'closedCaptionCueChanged',
 
+      /**
+       * Raised when asset dimensions become available.
+       *
+       * Provide the following arguments:
+       * <ul>
+       *   <li>width
```
Comment (angieisrani) [inline js/utils/constants.js]: string or number?

**Comment by rchalooyala** [comment]
added formatting

**Comment by angieisrani** [comment]
LGTM

**Pull request merged**